docs/en/faq.md

```diff
@@ -1,332 +1,169 @@
-
-1. If you want to make Spark's consumption of `Topic 1` keep up with its production speed, then you need `spark.executor.cores` * `spark.executor.instances` >= K / M
+If using Windows, enable SQL Server Agent (e.g., for SQL Server 2008):
+   - Refer to the [official documentation](https://learn.microsoft.com/en-us/previous-versions/sql/sql-server-2008-r2/ms191454(v=sql.105)).
+```
+Open "SQL Server Configuration Manager" from the Start menu, navigate to "SQL Server Services," right-click the "SQL Server Agent" instance, and start it.
+```
 
-2. When a data delay occurs, if you want the consumption speed not to be too fast, resulting in spark executor OOM, then you need to configure `spark.streaming.kafka.maxRatePerPartition` <= (`spark.executor.cores` * `spark.executor.instances`) * M / N
+3. Firstly, enable CDC at the database level:
+```
+USE TestDB; -- Replace with your actual database name
+EXEC sys.sp_cdc_enable_db;
 
-3. In general, both M and N are determined, and the conclusion can be drawn from 2: The size of `spark.streaming.kafka.maxRatePerPartition` is positively correlated with the size of `spark.executor.cores` * `spark.executor.instances`, and it can be increased while increasing the resource `maxRatePerPartition` to speed up consumption.
+-- Check if the database has CDC enabled
+SELECT name, is_cdc_enabled
+FROM sys.databases
+WHERE name = 'database'; -- Replace with the name of your database
+```
 
-![Kafka](../images/kafka.png)
+4. Secondly, enable CDC at the table level:
+```
+USE TestDB; -- Replace with your actual database name
+EXEC sys.sp_cdc_enable_table
+@source_schema = 'dbo',
+@source_name = 'table', -- Replace with the table name
+@role_name = NULL,
+@capture_instance = 'table'; -- Replace with a unique capture instance name
 
-## How can I solve the Error `Exception in thread "main" java.lang.NoSuchFieldError: INSTANCE`?
+-- Check if the table has CDC enabled
+SELECT name, is_tracked_by_cdc
+FROM sys.tables
+WHERE name = 'table'; -- Replace with the table name
+```
 
-The reason is that the version of httpclient.jar that comes with the CDH version of Spark is lower, and The httpclient version that ClickHouse JDBC is based on is 4.5.2, and the package versions conflict. The solution is to replace the jar package that comes with CDH with the httpclient-4.5.2 version.
+## Does SeaTunnel support CDC synchronization for tables without primary keys?
+No, CDC synchronization is not supported for tables without primary keys. This is because, if there are two identical rows upstream and one is deleted or modified, it would be impossible to distinguish which row should be deleted or modified downstream, potentially resulting in both rows being affected. 
 
-## The default JDK of my Spark cluster is JDK7. After I install JDK8, how can I specify that SeaTunnel starts with JDK8?
+## Error during PostgreSQL task execution: Caused by: org.postgresql.util.PSQLException: ERROR: all replication slots are in use
```

Comment:
It's strange to put connector related questions here, why not put them on the connector's own page.